v2/thirdpartyemailpassword/nestjs/guide.mdx

```diff
@@ -36,16 +36,16 @@ You can scaffold a module using the nest CLI by running this in the root folder
 ```bash
 nest g module auth
 ```
-The result should be a new `auth` folder with `auth.module.ts` in it. We should convert this into a [dynamic module](https://docs.nestjs.com/modules#dynamic-modules) so we can set parts of the SuperTokens configuration in the App module. Centralizing settings like this can be helpful for things like using a separate connection URI for testing.
+The result should be a new `auth` folder with `auth.module.ts` in it. 
 
 ### Add config type and injection token
 
-Add a `config.interface.ts` file next into the `auth` folder. We will put the type and injection token for the SuperTokens config here.
+Create a `config.interface.ts` file in the `auth` folder. We will put the type and injection token for the SuperTokens configuration here.
 
```

Comment:
since the time this guide was written, nestjs did not create sub folders for `auth/auth` for example. All the commands like `nest g service supertokens auth` or `nest g middleware auth auth` would create the files inside the `./src/auth` folder instead. So we would not have files like `./src/auth/auth/auth.filter.spec.ts`, instead, it was just `./src/auth/auth.filter.spec.ts`. 

Can you figure out how we can make it so that this extra folder is not created? It's really strange to have ./src/auth/auth/...

Reply:
The behavior I see, for example: `nest g middleware auth,` is that it knows it knows to put that file in the `auth` directory from the root (i.e., it does not need an extra `auth` in the command). When you add the extra `auth`, it nests it in `auth/auth`. So, I removed the double auth occurrences in the commands.

So the `auth` files are now located as such:

```txt
📦src
 ┣ 📂auth
 ┃ ┣ 📂supertokens
 ┃ ┃ ┣ 📜supertokens.service.spec.ts
 ┃ ┃ ┗ 📜supertokens.service.ts
 ┃ ┣ 📜auth.filter.spec.ts
 ┃ ┣ 📜auth.filter.ts
 ┃ ┣ 📜auth.guard.spec.ts
 ┃ ┣ 📜auth.guard.ts
 ┃ ┣ 📜auth.middleware.spec.ts
 ┃ ┣ 📜auth.middleware.ts
 ┃ ┣ 📜auth.module.ts
 ┃ ┗ 📜config.interface.ts
 ┣ 📜app.controller.spec.ts
 ┣ 📜app.controller.ts
 ┣ 📜app.module.ts
 ┣ 📜app.service.ts
 ┗ 📜main.ts
```